test/test_other.py

```diff
@@ -1307,6 +1307,41 @@ def test_export_all(self):
     self.emcc('lib.c', ['-Oz', '-sEXPORT_ALL', '-sLINKABLE', '--pre-js', 'main.js'], output_filename='a.out.js')
     self.assertContained('libf1\nlibf2\n', self.run_js('a.out.js'))
 
+  def test_minimal_runtime_export_all_modularize(self):
+    """
+    This test ensures that modularize and export all work simultaneously.
+
+    In addition, it ensures that EXPORT_ALL is honored while using MINIMAL_RUNTIME.
+    """
+
+    main = r'''
+      #include <stdio.h>
+      #include <emscripten.h>
+      EMSCRIPTEN_KEEPALIVE void libf1() { printf("libf1\n"); }
+      EMSCRIPTEN_KEEPALIVE void libf2() { printf("libf2\n"); }
+    '''
+    create_file('main.c', main)
```

Comment:
You can inline the string into `create_file`.  e.g. `create_file('main.c', '''...`